lib/containers/k8s.pm

```diff
@@ -64,7 +64,9 @@ sub install_k3s {
     assert_script_run("ln -s /etc/rancher/k3s/k3s.yaml ~/.kube/config");
     sleep 60;
     script_retry("kubectl get serviceaccount default -o name", delay => 60, retry => 3);
-    record_info('k3s', "k3s installed");
+    script_retry("kubectl get pods --all-namespaces", delay => 60, retry => 3);
```

Comment:
Why are we doing this? I assume it's to wait until k3s is ready. If so, please add a comment, e.g.

    # Await k3s to be ready
    script_retry( .... )

Reply:
done